Cargo.toml

```
@@ -45,10 +45,10 @@ members = [
 futures-core = { version = "0.3", default-features = false }
 futures-sink = { version = "0.3", default-features = false }
 futures-util = { version = "0.3", default-features = false }
-tokio-util = { version = "0.4.0", features = ["codec"] }
-tokio = { version = "0.3", features = ["io-util"] }
-bytes = "0.5.2"
-http = "0.2"
+tokio-util = { version = "0.5", features = ["codec"] }
+tokio = { version = "0.3.2", features = ["io-util"] }
+bytes = "0.6"
+http = { git = "https://github.com/paolobarbolini/http.git", branch = "bytes06" }
```

rather than adding a Git dependency on `http`, let's just bump it separately when the next version is published. `bytes` is a private dependency of `http`, so it's not an issue that it still uses 0.5.
```suggestion
http = "0.2"
```